libr/include/r_crypto/r_sm4.h

```diff
@@ -0,0 +1,24 @@
+#ifndef R_SM4_H
+#define R_SM4_H
+
+#include <r_util.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Constant Key CK */
+static const ut32 sm4_CK[32] = {
```

Comment:
move it into the C, thres no need to expose those constants globally

Reply:
I need them in `libr/crypto/p/crypto_sm4.c`